### ISSUE TITLE
Mixed content warnings

### ISSUE THREAD

**Issue by QuLogic** [ocefpaf/python4oceanographers#1]
GitHub Pages are redirecting to HTTPS by default now. However, almost all pages on this blog are showing [mixed-content warnings](https://support.mozilla.org/en-US/kb/mixed-content-blocking-firefox).

Most of these can be fixed by changing the CC 4.0 image that is linked by http: instead of https:.

**Comment by ocefpaf** [comment]
Thanks!  That is probably the side bar (the crowdfunding site) as well.

**Comment by QuLogic** [comment]
Some of it, but others are in the `HTML(html)` signature on all pages.

**Comment by ocefpaf** [comment]
I guess I fixed that in the latest commit.  There might be some other `http` here and there though.

PS: If you are interest I am moving away from pelican and the octopress theme for IPython notebook publishing.  Nikola seems like a better option now.

**Comment by QuLogic** [comment]
Looks pretty good on almost the entire first page of posts. Only [this one](https://ocefpaf.github.io/python4oceanographers/blog/2015/11/16/folium_quiver/) is broken, but I think it's because of the WMS tiles which unfortunately don't appear available over HTTPS.

**Comment by ocefpaf** [comment]
Is that on firefox?  I noticed that most of my folium examples are broken on firefox now, but I can see them fine using Chromium.

**Comment by QuLogic** [comment]
Yes it is. Actually, you're right, the folium examples don't work. I think it's because of this:

> The stylesheet https://raw.githubusercontent.com/python-visualization/folium/master/folium/templates/leaflet.awesome.rotate.css was not loaded because its MIME type, "text/plain", is not "text/css".

**Comment by ocefpaf** [comment]
Thanks!  I will take a look into that.  (It should be fixed upstream in folium.)
